packages/components/app/styles/components/button.scss

```diff
@@ -304,4 +309,24 @@ $size-props: (
       border-color: transparent;
     }
   }
+}
+
+
+// SPECIAL
+
+// we apply a visual treatment to alert the developer if a "href" HTML attribute is used (instead of the "@href" Ember argument)
```

Comment:
please let me know what you think of this

Reply:
seems like a good idea; if we end-up needing this in other places there may be value in converting it into a mixin